ext/src/zpages/CMakeLists.txt

```diff
@@ -0,0 +1,8 @@
+add_library(opentelemetry_zpages
+	tracez_processor.cc
+	../../include/opentelemetry/ext/zpages/tracez_processor.h)
+
+target_include_directories(opentelemetry_zpages PUBLIC ../../include)
+
+target_link_libraries(opentelemetry_zpages opentelemetry_api opentelemetry_trace)
+
```

Comment:
If the relative path here looks too strange, another option that compiled for me is adding the following lines to ext/CMakeLists.txt:
```
add_library(opentelemetry_zpages src/zpages/tracez_processor.cc include/opentelemetry/ext/zpages/tracez_processor.h)
target_include_directories(opentelemetry_zpages PUBLIC include PRIVATE src)
target_link_libraries(opentelemetry_zpages opentelemetry_api opentelemetry_trace)
```